apps/editing-toolkit/editing-toolkit-plugin/readme.txt

```diff
@@ -40,7 +40,10 @@ This plugin is experimental, so we don't provide any support for it outside of w
 
 == Changelog ==
 
-= 2.8.1 = 
+= 2.8.2 =
+* Gutenberg: Make line-height setting more robust. (https://github.com/Automattic/wp-calypso/pull/46873)
```

Comment:
Let's include that we actually fixed the line-height setting (through a different PR):

```suggestion
* Gutenberg: Support new line height setting (https://github.com/Automattic/wp-calypso/pull/46792)
* Gutenberg: Make line-height setting more robust. (https://github.com/Automattic/wp-calypso/pull/46873)
```